flang/test/Integration/OpenMP/map-types-and-sizes.f90

```diff
@@ -30,17 +30,17 @@ subroutine mapType_array
   !$omp end target
 end subroutine mapType_array
 
-!CHECK: @.offload_sizes{{.*}} = private unnamed_addr constant [3 x i64] [i64 0, i64 24, i64 4]
-!CHECK: @.offload_maptypes{{.*}} = private unnamed_addr constant [3 x i64] [i64 32, i64 281474976710657, i64 281474976711187]
+!CHECK: @.offload_sizes{{.*}} = private unnamed_addr constant [4 x i64] [i64 0, i64 24, i64 8, i64 4]
```

Comment:
can do, but it's generally data sizes corresponding to the type, so I am not too sure it'd be great to comment each individual one, there's a lot in this test and it may just add excess noise.

It will likely fail if you haven't got the corresponding lowering PR applied as well, which is part of the PR stack. Otherwise, the PR is stale about 3-4 weeks from sitting in review so that might be part of it as well! I'll address it when I update the PR if that is the case, thank you for pointing it out!